c/README.md

```diff
@@ -0,0 +1,49 @@
+# kuliya for C
+
+Algeria's college hierarchy dataset as a C library
+
+# Prerequisites
+
+- [CMake](https://cmake.org/download/) build system (minimum required version 3.19)
+- [Conan](https://conan.io/downloads) package manager
+- [Clang](https://clang.llvm.org/get_started.html) or [GCC](https://gcc.gnu.org/releases.html) compiler (on macOS clang comes with the developer tools)
+- Any IDE out there, preferably [VSCode](https://code.visualstudio.com/download) with the following extensions:
+  - [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)
+  - [CMake](https://marketplace.visualstudio.com/items?itemName=twxs.cmake)
+  - [CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools)
+  - [conan-extension](https://marketplace.visualstudio.com/items?itemName=konicy.conan-extension)
+
+# Get Started
+
+- Install build dependencies:
+  ```sh
+  conan install . --output-folder=build --build=missing
```

Comment:
```suggestion
  cd c && conan install . --output-folder=build --build=missing
```

Reply:
Since this `README.md` file was inside the `c/` folder, I thought I should not add the `cd` command but I think this is more clear